### DISCUSSION TITLE
How about write down retrospective note into Retrospective.md?

### DISCUSSION THREAD

**Discussion by jongfeel** [wisdom08/dev-community#13] [Ideas]
### Retrospective

회고라고 부르는 건데 다음 미팅 전까지 했던 내용에 대해서 느낀 점을 적어 보면 좋을 것 같습니다.
애자일 회고라고 검색하면 이것 저것 많이 나오는데
어떤 형식에 맞춰서 적기 보다는 생각나는 걸 적어 보면 좋겠습니다

### Process

그냥 issue에 적어도 되긴 하는데 그러면 나중에 클릭하면서 보기 ~힘들~귀찮을 수 있으니
README.md 파일에 Retrospective.md 파일에 링크를 두고 거기에 적어 보면 시간 순서에 따른 생각의 변화를 알 수 있게 됩니다.
물론 최신 글이 가장 위에 오도록 헤야 겠죠

#### Use pull request

역시 pull request 요청을 통해 내용을 적고 서로 확인하고
Retrospective.md 파일로 정리하는 걸로 하면 좋겠습니다

**Comment by wisdom08** [comment]
회고 내용을 작성했습니다. 지나치게 형식에 맞춰 작성을 하기는 했습니다..🙄
